schema/check_schemas.py

```diff
@@ -126,7 +126,8 @@ def main(args):
                       len(schema_errors), schema_count)
         for failure in schema_errors:
             logging.error('  %s', failure)
-        exit(1)
+        # We need to clobber the process
+        sys.exit(1)
     else:
         logging.info("All %d schema are valid in file %s", schema_count, output_filename)
         exit(0)
```

Comment:
You didn't change this line. But I wonder if you need it.

Reply:
I think sys.exit is appropriate here.